resources/helmfile/values/db-operator.values.yaml

```diff
@@ -1,8 +1,8 @@
 ---
-image: 
+image1:
```

Comment:
changed for a test?

Reply:
I'm switching between local and dev versions like that currently. I want to come up with a better solution, but don't have enough time atm